app/src/main/java/com/compscieddy/meetinthemiddle/model/Chat.java

```diff
@@ -63,7 +67,7 @@ public void update() {
     result.put("key", groupKey);
     result.put("userKey", userKey);
     result.put("chatMessage", chatMessage);
-    result.put("timestamp", timestamp);
+    result.put("currentUTCTime", currentUTCTime);
```

Comment:
Nice, I once forgot about updating this method once and was debugging for like 30 minutes 😒